### PR TITLE
docs: update links to point to juttle-engine and not outrigger

### PR DESCRIPTION
**Pull request by demmer**
Replace the links to the outrigger project with links to juttle-engine.

Note: this is to merge into the 0.4.x branch, to be included in a 0.4.1 release of juttle.

### CODE REVIEW

**Comment by demmer** [comment]
@dmehra @dmajda

**Comment by dmajda** [comment]
+1 for the changes

There are also few more mentions of Outrigger which I think should be fixed as part of this change:

```
CONTRIBUTING.md
39:- screenshot of the browser output, if the problem is with juttle-viz or outrigger

README.md
65:like [outrigger](#outrigger) so they are meant to be
68:For more end-to-end examples of juttle usage, see the [outrigger examples](https://github.com/juttle/outrigger/tree/master/examples).
149:<a name="outrigger"></a>
168:called [outrigger](https://github.com/juttle/outrigger).
```

There is also a mention in the Emacs mode, but this is probably something for a separate issue:

```
misc/emacs/juttle-derived-mode.el
17:(defvar juttle-run-prog "outrigger-client push --topic mydev --path")
```

**Comment by demmer** [comment]
@dmajda updated the rest of the mentions, please take a look.

@mccanne / @dmehra you guys might want to review the new language in the README that describes juttle-engine as opposed to outrigger.

File: README.md [hunk 1]
```diff
@@ -163,15 +163,18 @@ and energy), Juttle integrates these two layers
...
 of wiring Juttle dataflow computation to your browser-based views.
 
-We haven't yet worked out all of the details of this separation, but
-we've put together the components in a prototype test and development utility
-called [outrigger](https://github.com/juttle/outrigger).
-Outrigger pulls in the Juttle core from this repo along with
-the [juttle-viz](https://github.com/juttle/juttle-viz) visualization library and corresponding glue
-to implement Juttle's visualization capabilities.
-Outrigger lets you run Juttle programs from your local file system
+As an example, [Juttle Engine](https://github.com/juttle/juttle-engine) is an
```
Comment (dmajda) [inline README.md]: Ugh, a rebase killed my comment, so adding once more:

 “As an example” sounds a bit odd in the context of the previous paragraph. There we speak in quite firmly about the integration, but then we present Juttle Engine merely as an example.

Maybe something like “You can see this in action in Juttle Engine, our integrated environment…”?

**Comment by dmajda** [comment]
+1 beside my comment

File: README.md [hunk 2]
```diff
@@ -163,15 +163,18 @@ and energy), Juttle integrates these two layers
...
 of wiring Juttle dataflow computation to your browser-based views.
 
-We haven't yet worked out all of the details of this separation, but
-we've put together the components in a prototype test and development utility
-called [outrigger](https://github.com/juttle/outrigger).
-Outrigger pulls in the Juttle core from this repo along with
-the [juttle-viz](https://github.com/juttle/juttle-viz) visualization library and corresponding glue
-to implement Juttle's visualization capabilities.
-Outrigger lets you run Juttle programs from your local file system
+This can be done using [Juttle Engine](https://github.com/juttle/juttle-engine),
```
Comment (demmer) [inline README.md]: @dmajda Sorry about the rebase... I pushed quickly but not fast enough.

I updated the text to address your comments.

**Comment by mccanne** [comment]
How about mentioning that we envision Juttle Engine enabling other applications in addition to juttle-viewer in the future?

**Comment by demmer** [comment]
@mccanne good idea -- how's it look now?

**Comment by dmehra** [comment]
+1

**Pull request merged**